hyperspy/io_plugins/msa.py

```diff
@@ -216,34 +212,14 @@ def parse_msa_string(string, filename=None):
                     mapped.set_item(keywords[clean_par]['mapped_to'] +
                                     '_units', units)
 
-    # The data parameter needs some extra care
```

Comment:
Yes, I will revert this. I missed out the abbreviation of the month, which can be different using different localisation...